README.md

```diff
@@ -1,2 +1,40 @@
-# gaia-automation
-Charter for the Gaia Automation team
+Gaia Automation
+===============
+
+### Mission
+
+The Gaia Automation team takes responsibility for issues at the
+intersection of Firefox OS and automation. We aim to make it easy for
+developers of the FxOS system frontend and core FxOS apps to
+
+1. use external code,
+2. build gecko profiles for different scenarios (dev, test, prod),
+3. and test their code with unit and UI tests.
```

Comment:
this list seems out of order -- isn't #3 the first priority? it would also be good to see something more assertive/ambitious in the mission statement that reads like a clear commitment -- e.g. the quote I took from the meeting we had a while back and included in my note to the gaia manager "Ensure that tests are runnable by developers at all times. Tests should be easy to setup and execute."